examples/gatsby/README.md

```diff
@@ -2,7 +2,7 @@
   Using Builder.io with Gatsby
 </h1>
 
-> A demo for integrating [Builder.io](https://www.builder.io) with Gatsby using our GraphQL API and the `gatsby-source-graphql` plugin
+> A demo for integrating [Builder.io](https://www.builder.io) with Gatsby using our GraphQL API and our `@builder.io/gatsby` plugin
 
 This demo demonstrates creating dynamic pages in Builder.io on new URLs and generating them with Gatsby, as well
 as rendering specific parts of your site with Builder.io content via GraphQL queries (e.g. for pages, components, etc)
```

Comment:
theres a part of the readme I added below about see gatsby-node which is gone now

also lets link somewhere (there or otherwise) to the source of the gatsby plugin https://github.com/builderio/builder/tree/master/packages/gatsby